lms/static/scripts/ui-playground/components/ErrorComponents.js

```diff
@@ -42,29 +44,390 @@ function ErrorDialogExample() {
   }
 }
 
+function ErrorDialogAppExample({ errorCode = 'reused_consumer_key' }) {
+  const config = { api: {}, errorDialog: { errorCode } };
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  if (!dialogOpen) {
+    return (
+      <LabeledButton
+        onClick={() => setDialogOpen(!dialogOpen)}
+        variant="primary"
+      >
+        Show ErrorDialogApp Example
+      </LabeledButton>
+    );
+  } else {
+    return (
+      <Config.Provider value={config}>
+        <ErrorDialogApp />
+      </Config.Provider>
+    );
+  }
+}
+
+function OAuth2RedirectErrorAppExample({ errorCode = '' }) {
+  const config = {
+    api: {},
+    OAuth2RedirectError: {
+      errorCode,
+      canvasScopes: [
+        'This is a fake Canvas scope',
+        'These scopes would be',
+        'Written to the JS configuration object',
+        'By the back end in real life',
+      ],
+      errorDetails: fakeDetails,
+    },
+  };
+  const [dialogOpen, setDialogOpen] = useState(false);
+
+  if (!dialogOpen) {
+    return (
+      <LabeledButton
+        onClick={() => setDialogOpen(!dialogOpen)}
+        variant="primary"
+      >
+        Show OAuth2RedirectErrorApp Example
+      </LabeledButton>
+    );
+  } else {
+    return (
+      <Config.Provider value={config}>
+        <OAuth2RedirectErrorApp />
+      </Config.Provider>
+    );
+  }
+}
+
 export default function ErrorComponents() {
   return (
     <Library.Page title="Errors">
+      <div className="LMSLibrary__content">
+        <h2>Intro</h2>
```

Comment:
This is an existing issue unrelated to the changes in this PR, but the h1, h2 and h3 heading sizes are all rather close. I found this made it a little tricky to grok the overall structure of this page.

Reply:
Agreed!